packages/rrweb/src/replay/index.ts

```diff
@@ -1219,6 +1219,14 @@ export class Replayer {
             if (isSync) {
               if (d.type === MouseInteractions.TouchStart) {
                 pointer.touchActive = true;
+
+                // prevents multiple touch circles from staying on the screen
+                // when the user seeks by breadcrumbs
+                Object.values(this.pointers).forEach((p) => {
```

Comment:
do we have unit tests for this in this behavior in this lib already?

Reply:
currently fixing existing tests. may add new ones